src/Raven.Server/Documents/Indexes/Static/StaticIndexBase.cs

```diff
@@ -200,13 +200,13 @@ public abstract class AbstractStaticIndexBase
 
         public CompiledIndexField[] GroupByFields;
 
-        private HashSet<string> _groupByFieldNames;
+        private List<string> _groupByFieldNames;
 
-        public HashSet<string> GroupByFieldNames
+        public List<string> GroupByFieldNames
```

Comment:
Same comment as above

Reply:
I meant this one - https://github.com/ravendb/ravendb/pull/15936#discussion_r1138709370